tests/integration/__init__.py

```diff
@@ -0,0 +1,59 @@
+import luigi
+import os
+import pandas as pd
+import sqlalchemy
+import sys
+if sys.version_info[0] < 3:
+    from StringIO import StringIO
+else:
+    from io import StringIO
+
+from .. import test_data
+
+
+POSTGRES_HOST = os.environ.get('TEST_POSTGRES_HOST', 'localhost')
+POSTGRES_USER = os.environ.get('TEST_POSTGRES_USER', 'postgres')
+POSTGRES_PASSWORD = os.environ.get('TEST_POSTGRES_PASSWORD', '')
+POSTGRES_DATABASE = os.environ.get('TEST_POSTGRES_DATABASE', 'test_db')
+
+
+def setup_package():
+    engine = sqlalchemy.create_engine(
+        'postgresql://{user}:{password}@{host}/{dbname}'.format(
+            user=POSTGRES_USER, password=POSTGRES_PASSWORD,
+            host=POSTGRES_HOST, dbname=POSTGRES_DATABASE
+        )
+    )
+
+    test_config = luigi.configuration.LuigiConfigParser.instance()
+    test_config.set('QueryPostgres', 'host', POSTGRES_HOST)
+    test_config.set('QueryPostgres', 'database', POSTGRES_DATABASE)
+    test_config.set('QueryPostgres', 'user', POSTGRES_USER)
+    test_config.set('QueryPostgres', 'password', POSTGRES_PASSWORD)
+
+    # meta = sqlalchemy.MetaData(engine)
```

Comment:
Is this going to be used. If not I'm a fan of just removing it

Reply:
Should be gone now.